blockbase/block-template-parts/header.html

```diff
@@ -1,14 +1,19 @@
-<!-- wp:group {"className":"site-header","layout":{"type":"flex"},"style":{"spacing":{"padding":{"bottom":"80px"}}}} -->
-<div class="wp-block-group site-header" style="padding-bottom:80px">
-	<!-- wp:site-logo /-->
+<!-- wp:group {"className":"site-header","layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"padding":{"bottom":"80px","top":"var(--wp--custom--gap--vertical)"}}}} -->
+<div class="wp-block-group site-header" style="padding-bottom:80px;padding-top:var(--wp--custom--gap--vertical)">
```

Comment:
I'm hesitant to use variables in this spot.  It doesn't translate well into the FSE.

Reply:
What if we had a fallback value?

Reply:
In the future the value won't be blank in the editor, but it will let the user know that the value is dynamic. It's a bit obscure but it's the way the editor is meant to work, so I think it's the best option for us too, since the editor will assume that this is a thing that themes will want to do in some places.

Reply:
Yeah, we're using variables this way in Twenty Twenty-Two, and I think it's ok as long as there's a fallback for when/if users switch themes.